datatypes/src/primitives/datetime.rs

```diff
@@ -323,6 +323,12 @@ impl From<chrono::DateTime<chrono::Utc>> for DateTime {
     }
 }
 
+impl From<DateTime> for chrono::DateTime<chrono::Utc> {
+    fn from(datetime: DateTime) -> Self {
+        datetime.datetime
+    }
+}
+
```

Comment:
test coverage is below 70%

Reply:
I cannot really comprehend how this is calculated if I look at the report. I mean I touched all of the operators so maybe that leads to this measurement? But it also reports `Measurement`s to be untested now, and I didn't touch any of that.  https://coveralls.io/builds/71398158/source?filename=services%2Fsrc%2Fapi%2Fmodel%2Fdatatypes.rs#L765

If you want anything specific to be tested just let me know.